src/database/models/Report.js

```diff
@@ -0,0 +1,48 @@
+export default (sequelize, DataTypes) => {
+  const Report = sequelize.define('Report', {
+    reporterId: {
+      type: DataTypes.INTEGER,
+      allowNull: false
+    },
+    articleId: {
+      type: DataTypes.INTEGER,
+      allowNull: false
+    },
+    violation: {
+      type: DataTypes.ENUM(
+        'Discrimination',
+        'Plagiarism',
+        'Sexual Content',
+        'OtherReport',
+        'Offensive Language'
+      ),
+      allowNull: false
+    },
+    description: {
+      type: DataTypes.STRING,
+      allowNull: false
```

Comment:
allow null values for description.

However, in your business logic validate that description is provided if violation  === 'others'.